packages/manager/core/generator/app/conditional-templates/listing/v2/index.tsx.hbs

```diff
@@ -87,7 +87,7 @@ export default function Listing() {
   }
 
   if (isLoading && !flattenData) {
-    return (<div><Loading/></div>)
+    return (<div data-testid="listing-page-spinner"><Loading/></div>)
   }
```

Comment:
Just for information: On ODS 18 to fix for remove <div></div> container. We need add "inline" on Loading spinner. 

I don't known if it's relevant here :)